src/App.js

```diff
@@ -1,12 +1,13 @@
-import React, { Component } from 'react';
-import CreatureList from './components/CreatureList';
+import React, {Component} from 'react';
+import {CreatureList, StateManagement} from './components';
 import './App.css';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <CreatureList />
+        <StateManagement/>
```

Comment:
In the worlds of Max Eddy, "What point does this whitespace serve?" in regards to every piece of whitespace I ever put because of it was "preferred"

Shrug. PHPStorm deletes the space.